rootfs/bin/restore_cache

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+cache_file=/tmp/cache.tgz
+# shellcheck disable=SC1091
+source ./normalize_storage
```

Comment:
I'd try the full path `/bin/normalize_storage`.

because of the error: `./normalize_storage: No such file or directory`